api/rs/slint/android.rs

```diff
@@ -46,19 +46,35 @@
 //!
 //! ## Building and Deploying
 //!
+//! Building a Rust application requires the target toolchain to be installed. You can install it via *rustup*:
+//!
+//! ```sh
+//! rustup target add aarch64-linux-android
+//! ```
+//!
+//! Please ensure that you have the Android NDK and SDK installed and properly set up in your development environment for the above command to work as expected.
```

Comment:
Slight simpler wording, but also the removal of "above command". The `rustup` call does not require the Android NDK or SDK.

```suggestion
//! Make sure that you have the Android NDK and SDK installed and set up in your development environment.
```